x-pack/solutions/security/plugins/security_solution/public/data_view_manager/hooks/use_init_data_view_manager.test.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { renderHook } from '@testing-library/react';
+import { TestProviders } from '../../common/mock';
+import { useInitDataViewManager } from './use_init_data_view_manager';
+
+describe('useInitDataViewPicker', () => {
+  it('should render', () => {
```

Comment:
I think you missed an assertion here? Or are you just making sure no errors are thrown?